neural_speed/core/layers/Ops.h

```diff
@@ -48,6 +48,7 @@ enum ne_op {
 
   NE_OP_MUL_MAT,
   NE_OP_MUL_MAT_BIAS,
+  NE_OP_MUL_MAT_ID,
```

Comment:
what the difference between NE_OP_MUL_MAT_ID and NE_OP_MUL_MAT

Reply:
NE_OP_MUL_MAT_ID has more input tensor, selected expert id and 8 expert weights, different tokens use the expert id to select different expert weights for calculation.